lib/reactions/fiat_rate_update.dart

```diff
@@ -29,12 +29,6 @@ Future<void> startFiatRateUpdate(
 
       if (appStore.wallet!.type == WalletType.haven) {
         await updateHavenRate(fiatConversionStore);
-      } else {
-        fiatConversionStore.prices[appStore.wallet!.currency] =
-            await FiatConversionService.fetchPrice(
-                crypto: appStore.wallet!.currency,
-                fiat: settingsStore.fiatCurrency,
-                torOnly: settingsStore.fiatApiMode == FiatApiMode.torOnly);
```

Comment:
why did you remove this?
like this, any other wallet will not have its fiat rate

Reply:
I didn't finish cleaning it up, see the changes I just pushed